src/docs-style-guide.rst

```diff
@@ -661,8 +661,12 @@ In these cases, you may want to use the :rst:dir:`rubric` directive.
 
       errors = []
       sym_list = ['===','---','~~~','"""','\'\'\'']
+      cnt_label = 0
```

Comment:
I'm merging. But, just for your own edification, here is what I would have done:

```
is_doc_title = True

for matchobj in re.finditer(regex,text):
    if is_doc_title:
        is_doc_title = False
        continue
```

This avoids keeping an ever-increasing integer for no reason (a very minor efficiency) and is also more explicitly a coded version of what is actually happening.

Reply:
Thank you for the suggestion! I will add this as a fix to improve the efficiency!